cdap-master/src/main/java/co/cask/cdap/data/tools/UpgradeDatasetServiceManager.java

```diff
@@ -148,7 +155,14 @@ Injector createInjector(CConfiguration cConf, Configuration hConf) {
       new RemoteSystemOperationsServiceModule(),
       new SecureStoreModules().getDistributedModules(),
       new AuthorizationModule(),
-      new AuthorizationEnforcementModule().getMasterModule(),
+      // we override the AuthorizationEnforcementService from the one which has been started in the upgrade tool to
```

Comment:
can you reference the jira to remove the ``DatasetServiceManager`` injector here too?